### PR TITLE
[stable10] Use proper casing of username in avatar filesystem setup 

### PR DESCRIPTION
**Pull request by MorrisJobke**
Backport of #1595

cc @nickvergessen @rullzer @icewind1991 @LukasReschke

### CODE REVIEW

File: lib/private/AvatarManager.php [hunk 1]
```diff
@@ -85,11 +85,19 @@ public function getAvatar($userId) {
...
+
 		/*
 		 * Fix for #22119
-		 * Basically we do not want to copy the skeleton folder
+		 * Basically we do not want to copy the skeleton folder.
+		 *
+		 * For unit test purposes this is ignored when run in PHPUnit.
 		 */
-		\OC\Files\Filesystem::initMountPoints($userId);
+		if(!defined('PHPUNIT_RUN')) {
```
Comment (nickvergessen) [inline lib/private/AvatarManager.php]: this is so wrong!

Comment (MorrisJobke) [inline lib/private/AvatarManager.php]: I removed the commit of @LukasReschke

Comment (MorrisJobke) [inline lib/private/AvatarManager.php]: It only changed the tests implementation.

**Comment by rullzer** [comment]
LGTM

**Comment by MorrisJobke** [comment]
@nickvergessen Okay to merge?

**Comment by nickvergessen** [comment]
👍 

Can you also take care of the master PR?

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**